packages/docs/components/landing/Features.vue

```diff
@@ -29,6 +29,7 @@
           >
             <img
               :src="icon"
+              :alt="description"
               aria-hidden="true"
```

Comment:
```suggestion
              alt=""
```

It is pure decorative image without any meaning. The same description text is displayed bellow in paragraph. For such images, it is better to have empty alt text (see https://www.w3.org/WAI/tutorials/images/decorative/) or any alt text (like "decorative image") with `aria-hidden`.